src/test/java/duke/util/TaskListTest.java

```diff
@@ -0,0 +1,107 @@
+package duke.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.ToDo;
+
+class TaskListTest {
+    // @author Jonathan Cook
+    // Reused from https://www.baeldung.com/java-testing-system-out-println
+    // with minor modifications
+    private final PrintStream standardOut = System.out;
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
```

Comment:
You might want to consider changing the variable names to `STANDARD_OUT` and `OUTPUT_STREAM_CAPTOR`? Since they are declared as being final.